Pod/Classes/VOKCoreDataManager.m

```diff
@@ -296,6 +327,38 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
     return [returnArray copy];
 }
 
+- (NSArray *)arrayToImportAfterImportingRepeatedUniqueKeys:(NSArray *)repeatedUniqueKeys
+                                   forForeignUniqueKeyPath:(NSString *)foreignUniqueKeyPath
+                                           usingInputArray:(NSArray *)inputArray
+                                                  forClass:(Class)objectClass
+                                               withContext:(NSManagedObjectContext *)contextOrNil
+{
+    NSMutableArray *uniquedInputArray = [NSMutableArray arrayWithArray:inputArray];
+    
+    NSMutableArray *itemsToRemove = [NSMutableArray array];
+    for (id repeatedKey in repeatedUniqueKeys) {
+        for (id item in inputArray) {
```

Comment:
Ohh... I think I see why and now the comment on line 352–3 makes some sense.  Hrm.